### PR TITLE
backend/eth: handle connection error

### PR DESCRIPTION
**Pull request by benma**
Set account to offline, reconnect automatically in next poll
iteration.

### CODE REVIEW

File: backend/coins/eth/account.go [hunk 1]
```diff
@@ -42,6 +42,8 @@ type Account struct {
 	getSigningConfiguration func() (*signing.Configuration, error)
 	signingConfiguration    *signing.Configuration
 	keystores               keystore.Keystores
+	offline                 bool
+	onEvent                 func(Event)
```
Comment (KasparEtter) [inline backend/coins/eth/account.go]: You could have used the observable package.

File: backend/coins/eth/coin.go [hunk 2]
```diff
@@ -59,7 +59,7 @@ func (coin *Coin) Initialize() {
 		client, err := ethclient.Dial(coin.nodeURL)
 		if err != nil {
 			// TODO: init conn lazily, feed error via EventStatusChanged
-			panic(err)
+			panic("LOL")
```
Comment (KasparEtter) [inline backend/coins/eth/coin.go]: Why does a "LOL" make it to the master?

File: backend/coins/eth/account.go (hunk 1 above)
Comment (benma) [reply backend/coins/eth/account.go]: Using the same as btc/account.go, so we can refactor both at once.

File: backend/coins/eth/coin.go (hunk 2 above)
Comment (benma) [reply backend/coins/eth/coin.go]: :scream:

**Pull request merged**